engine/schema/src/main/java/com/cloud/configuration/dao/ResourceCountDaoImpl.java

```diff
@@ -266,18 +266,18 @@ public long countMemoryAllocatedToAccount(long accountId) {
     }
 
     private long executeSqlCountComputingResourcesForAccount(long accountId, String sqlCountComputingResourcesAllocatedToAccount) {
-        try (TransactionLegacy tx = TransactionLegacy.currentTxn()) {
+        TransactionLegacy tx = TransactionLegacy.currentTxn()
```

Comment:
@DaanHoogland  you missed ";" at the end of line 269.

Reply:
fixed